blockbase/theme.json

```diff
@@ -347,197 +347,6 @@
 					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
 					"slug": "system-font",
 					"name": "System Font"
-				},
-				{
-					"fontFamily": "Arvo, serif",
-					"slug": "arvo",
-					"name": "Arvo",
-					"provider": "blockbase-fonts"
-				},
-				{
-					"fontFamily": "'Bodoni Moda', serif",
-					"slug": "bodoni-moda",
-					"name": "Bodoni Moda",
-					"provider": "blockbase-fonts"
-				},
-				{
-					"fontFamily": "Cabin, sans-serif",
-					"slug": "cabin",
-					"name": "Cabin",
-					"provider": "blockbase-fonts"
-				},
-				{
-					"fontFamily": "Chivo, sans-serif",
-					"slug": "chivo",
-					"name": "Chivo",
-					"provider": "blockbase-fonts"
-				},
-				{
-					"fontFamily": "'Courier Prime', serif",
-					"slug": "courier-prime",
-					"name": "Courier Prime",
-					"provider": "blockbase-fonts"
-				},
-				{
-					"fontFamily": "'DM Sans', sans-serif",
-					"slug": "dm-sans",
-					"name": "DM Sans",
-					"provider": "blockbase-fonts"
-				},
-				{
-					"fontFamily": "Domine, serif",
-					"slug": "domine",
-					"name": "Domine",
-					"provider": "blockbase-fonts"
-				},
-				{
-					"fontFamily": "'EB Garamond', serif",
-					"slug": "eb-garamond",
-					"name": "EB Garamond",
-					"provider": "blockbase-fonts"
-				},
-				{
-					"fontFamily": "'Fira Sans', sans-serif",
-					"slug": "fira-sans",
-					"name": "Fira Sans",
-					"provider": "blockbase-fonts"
-				},
-				{
-					"fontFamily": "'Helvetica Neue','Helvetica', 'Arial', sans-serif",
```

Comment:
Since these Helvetica+Arial fonts are from the OS, should you keep them here?

Reply:
Oh yeah, we can do that.